typescript/scripts/eval/promptfoo/run_aiconfig.ts

```diff
@@ -0,0 +1,33 @@
+import { config as load_dotenv } from "dotenv";
```

Comment:
I don't think dotenv is a dependency inside of `typescript/package.json` so this will have an error when it hits main now.

To test this you would need to `cd ~/Projects/aiconfig && rm -rf node_modules`, then try running this again since your previous commit version had installed dotenv.

Fix would be add dotenv as dependency to `typescript/package.json`, run `yarn` in the typescript directory & add `package.json` and `yarn.lock` to this diff.

Reply:
Removed import.